### PR TITLE
CodeJam 2023

### PR DESCRIPTION
**Pull request by LimaoC**
Add CodeJam 2023 competition page to the website. Still waiting on prize amounts to be confirmed, so I've left those as TBD for now.

### CODE REVIEW

File: content/competitions/codejam-2023.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                trading firms and a key market maker in various products listed on exchanges throughout the world.
+                They'll help support your journey and also provide you with some amazing networking opportunities!
+            </p>
+            <p>Keep reading to find out more about the competition and what to expect. If you have any more questions,
+                you can reach out at any of our social media pages or via our lovely
+                <a href="https://discord.uqcs.org">Discord community</a>.
+            </p>
+
+            <h3>So, how does it all work?</h3>
+            <p>The CodeJam is programming competition hosted online through
```
Comment (RachelChiong) [inline content/competitions/codejam-2023.html:40]: Change to "CodeJam is a programming competition..."

File: content/competitions/codejam-2023.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            <h3>What do I do on the day?</h3>
+            <p>The CodeJam will run from 10am – 4pm on Saturday 1st April. The live event will be hosted from the
+                Advanced Engineering Building, room 49-301, but you can participate from anywhere. However, you must be competing
+                in-person to be eligible for prizes. You can join in at any time but the more time you spend, the better your
+                chances are of winning prizes!
+            </p>
+            <p>We'll also have free lunch on the day too, so be sure not to miss out!</p>
+            <p>We’ll post updates and reminders throughout the day on our <a href="http://discord.uqcs.org/">Discord
+                    server</a>.
+                Both the event kickoff and presentation of prizes will be live streamed on our
```
Comment (RachelChiong) [inline content/competitions/codejam-2023.html:78]: Aside: should we live stream these again this year? as there probs wont be many people competing online given the new rules

**Review by RachelChiong — CHANGES_REQUESTED** [review]
See comment(s). Otherwise lgtm

File: content/competitions/codejam-2023.html (hunk 2 above)
Comment (LimaoC) [reply content/competitions/codejam-2023.html:78]: I think they'd be good to livestream just to serve as a memento on the YouTube channel especially the prize presentations

File: content/competitions/codejam-2023.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+title: "CodeJam 2023"
+date: 2022-03-14T19:13:14+10:00
```
Comment (JamesDearlove) [inline content/competitions/codejam-2023.html:3]: Nit: Date should be from this year

**Review by JamesDearlove — APPROVED** [review]
Just a nit, otherwise lgtm

File: content/competitions/codejam-2023.html (hunk 3 above)
Comment (LimaoC) [reply content/competitions/codejam-2023.html:3]: See I just copy & paste the `html` file from the previous year so I have to change this manually, guess I forgot to change the date :joy:

**Review by RachelChiong — APPROVED** [review]

**Pull request merged**